Makefile

```diff
@@ -1,4 +1,16 @@
 .PHONY: dev
 
+compose:
+	@docker-compose up -d || docker compose up -d
+
+deployer:
+	@make watch ; go run ./deployer-service
+
+uploader:
+	@make watch ; go run ./uploader-service
+
+watch:
+	@find . -name "*.go" | entr -r sh -c 'echo "Restarted"'
+
 dev:
-	@find ./$(service) ./lib ./config -name "*.go" | entr -r sh -c 'echo "Restarted" && go run ./$(service)'
\ No newline at end of file
+	@make compose ; make deployer ; make uploader
```

Comment:
That's looking good, one more request change here.

The stdout of the services are not being logged, When you use `;`, it runs the commands in sequence, but it doesn't propagate the output of the second command if the first command is still running.